### PR TITLE
add CITATION.cff

### PR DESCRIPTION
**Pull request by Aariq**
This draft PR adds a CITATION.cff file. This file gets used by GitHub to populate a little "Cite This Repository" button and is used by Zenodo to populate archive metadata.  Please help by filling out your info in it including ORCID if you have one and affiliation.  See Kristina's entry for an example of formatting.  You do not need to include your email, as I think it makes sense to only include one.

Questions/comments:

- The only allowed citation types are "software" or "data"
- BUT, we can use a `preferred-citation:` that can be of any type, e.g. "book".  That will override the software citation for GitHub's "cite this" button, but I don't think Zenodo uses that information.  Maybe keeping the type as software is fine.
- I'm not sure if it is most appropriate to use your current affiliation or you affiliation _at the time of contribution_.  Would love opinions on this
- Authors are in order of: Kristina, authors from OSF citation, new members in order of joining the group.  Happy to take suggestions on a different order.  I figure leader first, every one else in order of contribution is probably fine?
- We can include a `version:` in CITATION.cff but for this book it seemed more appropriate to just update the `date-released` every time we archive a snapshot of the repo.

Still TODO:
- [ ] Add citation info to _quarto.yml

After this PR is merged, I'll link to Zenodo, make a release, and then add the new DOI to CITATION.cff and _quarto.yml

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+authors:
+  - given-names: Kristina
+    family-names: Riemer
+    affiliation: >-
+      University of Arizona, Communications & Cyber
+      Technologies Data Science
+    email: kristinariemer@arizona.edu
+    orcid: 'https://orcid.org/0000-0003-3802-3331'
+  - given-names: David S.
+    family-names: LeBauer
```
Comment (dlebauer) [inline CITATION.cff:20]:
```suggestion
    family-names: LeBauer
    affiliation: University of Arizona
    orcid: 'https://orcid.org/0000-0001-7228-053X'
```

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    affiliation: University of Arizona
```
Comment (dlebauer) [inline CITATION.cff:21]:
```suggestion
    affiliation: >-
      University of Arizona, Communications & Cyber
      Technologies Data Science
```

**Comment by dlebauer** [comment]
Thanks for doing this!
- Can't the type be 'manual'? https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files#citing-something-other-than-software
- Regarding authorship: 
  - I agree it makes sense for Kristina to be first author as she has taken leadership of the group and is responsible for how it works.
  - I don't think it makes sense to keep the original order (me then alphabetical) and then switch to the order of joining the group; I like the idea of order of joining the group. I'd propose [Kristina, order of the group, David]. Maybe add a comment to the CFF with the protocol.

**Comment by Aariq** [comment]
> Can't the type be 'manual'?

Not according to `cffr::cff_validate()` or the [cffinit tool](https://citation-file-format.github.io/cff-initializer-javascript/#/)

> I don't think it makes sense to keep the original order (me then alphabetical) and then switch to the order of joining the group; I like the idea of order of joining the group. I'd propose [Kristina, order of joining group, David].

Sounds great to me, but I don't know the order.  Could you suggest a change in the PR?

> plz add a comment to the CFF with the protocol + link to this PR

Good idea.  I'll add a note to the `message:` field as well as a comment linking to this discussion.

**Comment by dlebauer** [comment]
PR

**Comment by Aariq** [comment]
`type: manual` is only valid for `preferred-citation`, not the main citation type where only `software` and `data` are allowed. don't think that `preferred-citation` is used for the primary citation metadata by Zenodo the way it is by GitHub. Not sure though.

**Review by dlebauer — APPROVED** [review]

File: CITATION.cff [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+      Universtiy of Arizona, Communications & Cyber
+      Technologies Data Science
+  - given-names: Julian
+    family-names: Pistorius
+  - given-names: Jorge. E.
+    family-names: Barrios
+  - given-names: Emily. J.
+    family-names: Cain
+  - given-names: Jessica
+    family-names: Guo
```
Comment (Aariq) [inline CITATION.cff:34]:
```suggestion
  - given-names: Jessica
    family-names: Guo
    orcid: 'https://orcid.org/0000-0002-9566-9182'
```

File: CITATION.cff [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    family-names: Scott
+    affiliation: >-
+      University of Arizona, Communications & Cyber
+      Technologies Data Science
+    orcid: 'https://orcid.org/0000-0002-7430-7879'
+  - given-names: Renata
+    family-names: Diaz
+    affiliation: >-
+      Universtiy of Arizona, Communications & Cyber
+      Technologies Data Science
```
Comment (Aariq) [inline CITATION.cff:46]:
```suggestion
  - given-names: Renata
    family-names: Diaz
    orcid: 'https://orcid.org/0000-0003-0803-4734'
    affiliation: >-
      Universtiy of Arizona, Communications & Cyber
      Technologies Data Science
```

**Comment by Aariq** [comment]
I'll check in with folks at standup and try to get this merged today or tomorrow and I'll open a follow-up issue for adding metadata to _quarto.yml

**Pull request merged**